### PR TITLE
Submodules revision

### PR DESCRIPTION
**Pull request by Budalebah**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [submodule "unicorn"]
-	path = unicorn
-	url = git@github.com:geohot/unicorn.git
+    path = /workspace/activities/opml/unicorn
+    url = https://github.com/geohot/unicorn.git
 [submodule "mlgo"]
-	path = mlgo
-	url = git@github.com:OPML-Labs/mlgo.git
+    path = /workspace/activities/opml/mlgo
+    url = https://github.com/someuser/mlgo.git
```
Comment (0x1cc) [inline .gitmodules:6]: "someuser"?

**Comment by 0x1cc** [comment]
Thank you for your contribution to opML. But what is the purpose of modifying git submodule from "ssh" to "https"?

**Comment by Budalebah** [comment]
when I use ssh it can't pull the data

**Comment by Budalebah** [comment]
https://github.com/Budalebah/ora-io-discord-gif.git and also i preaped a gif pack for dc

**Comment by 0x1cc** [comment]
> when I use ssh it can't pull the data

It may be due to the configuration of git. Anyway, thanks for your interests. I will close this PR. If you have any other questions, please raise an issue.

**Pull request closed without merging**